contents/staticAnalysis/PMD.md

```diff
@@ -1,5 +1,5 @@
 ## PMD
-PMD is a static analyser for Java, JavaScript, Salesforce.com Apex, PL/SQL, Apache Velocity, XML, XSL. The Copy-pate-detector ([CPD Find duplicated code](http://pmd.sourceforge.net/pmd-4.3.0/cpd.html)) is also included as an add-on in PMD.
+PMD is a static analyser for Java, JavaScript, Salesforce.com Apex, PL/SQL, Apache Velocity, XML, XSL. The copy/paste-detector([CPD](http://pmd.sourceforge.net/pmd-4.3.0/cpd.html)), which helps to find duplicated code is also included as an add-on in PMD.
```

Comment:
change ", which helps to find duplicated code" to "which helps to find duplicated code,"